nifi-commons/nifi-properties/src/main/java/org/apache/nifi/util/NiFiProperties.java

```diff
@@ -1335,6 +1335,13 @@ public boolean isZooKeeperClientSecure() {
         return Boolean.parseBoolean(clientSecure);
     }
 
+    public boolean isZooKeeperTlsConfigurationPresent() {
```

Comment:
I think you're right, I should check that the property is present. I think I initially had it in there and a separate method to check if it was true/false. I guess I decided it might be better to check config is present and also check if we want it to be secure or not - I think that's wrong though and I will change it to check that the property is present.

I think the concept should be that TLS + ZK is all or nothing. If a user sets ZOOKEEPER_CLIENT_SECURE=TRUE, they should expect all aspects of ZK should be secured by TLS.